js/obj/sim/SimulatorDOM.js

```diff
@@ -11,16 +11,17 @@
 *
 * The "parent" Simulator instance is passed in as a parameter for the constructor,
 * and be accessed internally by `this.sim`.
-*
+*@example
+* var mainSim = new Simulator();
+* var dom = new SimulatorDOM(mainSim);
```

Comment:
Actually, this line is not necessary for using SimuatorDOM. When mainSim is created as a new Simulator object instance, its constructor will create a member variable (`this.dom` when accessed within the object instance, or `mainSim.dom` when accessed from the outside).

This member variable is automatically initialized to contain a new object instance of SimulatorDOM, so you don't need to do it manually. 

In the example that you wrote (and in the next few examples for SimulatorDOM.js), `dom` and `mainSim.dom` are different, unrelated instances of the SimulatorDOM class.